src/server/terrain/biomes.zig

```diff
@@ -289,6 +289,7 @@ pub const Biome = struct { // MARK: Biome
 	caves: f32,
 	caveRadiusFactor: f32,
 	crystals: u32,
+	soilCreep: f32, // How much of the surface structure should be eroded depending on the slope.
```

Comment:
Should a doc comment (triple `/`) in the line before the identifier, then it will also show up in ZLS. I think